### PR TITLE
Added test for the EarthquakeConsumer

### PR DESCRIPTION
**Pull request by orpiske**

### CODE REVIEW

**Comment by orpiske** [comment]
@bouskaJ wdyt?

File: test/EarthquakeConsumer.feature [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        "alert": "@assertThat(anyOf(nullValue(), notNullValue()))@",
+        "status": "@assertThat(notNullValue())@",
+        "tsunami": "@assertThat(notNullValue())@",
+        "sig": "@assertThat(notNullValue())@",
+        "net": "@assertThat(notNullValue())@",
+        "code": "@assertThat(notNullValue())@",
+        "ids": "@assertThat(notNullValue())@",
+        "sources": "@assertThat(notNullValue())@",
+        "types": "@assertThat(notNullValue())@",
+        "nst": "@assertThat(anyOf(nullValue(), notNullValue()))@",
```
Comment (bouskaJ) [inline test/EarthquakeConsumer.feature]: You can use `@ignore@`

File: test/EarthquakeConsumer.feature [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        "rms": "@assertThat(notNullValue())@",
+        "gap": "@assertThat(anyOf(nullValue(), notNullValue()))@",
```
Comment (bouskaJ) [inline test/EarthquakeConsumer.feature]: `@ignore@`

File: test/EarthquakeConsumer.feature [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        "place": "@assertThat(notNullValue())@",
+        "time": "@assertThat(notNullValue())@",
+        "updated": "@assertThat(notNullValue())@",
+        "tz": "@assertThat(notNullValue())@",
+        "url": "@assertThat(notNullValue())@",
+        "detail": "@assertThat(notNullValue())@",
+        "felt": "@assertThat(notNullValue())@",
+        "cdi": "@assertThat(notNullValue())@",
+        "mmi": "@assertThat(notNullValue())@",
+        "alert": "@assertThat(anyOf(nullValue(), notNullValue()))@",
```
Comment (bouskaJ) [inline test/EarthquakeConsumer.feature]: @ignore@

File: test/EarthquakeConsumer.feature [hunk 4]
```diff
@@ -0,0 +1,48 @@
+Feature: Earthquake consumer test
+
+  Background:
+    Given Kafka connection
+        | url       | event-streaming-kafka-cluster-kafka-brokers.event-streaming-kafka-cluster:9094 |
```
Comment (bouskaJ) [inline test/EarthquakeConsumer.feature]: Use `event-streaming-kafka-cluster-kafka-bootstrap.event-streaming-kafka-cluster:9092`

**Comment by orpiske** [comment]
Thanks for the review @bouskaJ

**Pull request merged**